load/loader.py

```diff
@@ -168,7 +168,7 @@ def load_data(opts, conf):
     if opts.task != 'dates' and opts.state is None:
         raise Exception('--state is required for tasks other than "dates"')
 
-    opts.state = opts.state.lower()
+    opts.state = opts.state.lower() if opts.state   is not None else None
```

Comment:
This is small, but there are some extra spaces here that aren't checked by pylint. Once those are gone I think this is good to go!